rpc-client/src/subcommands/blockchain_subcommands.rs

```diff
@@ -91,6 +91,14 @@ pub enum BlockchainCommand {
         /// The address to query by.
         address: Address,
 
```

Comment:
@sisou Here is where we were facing the issues. 

Maybe for now, we could add the object as you mentioned in the RPC request, but ignore those filters in the cli in the #L296